### ISSUE TITLE
Use cDNA to polish exons

### ISSUE THREAD

**Issue by SalvadorGJ** [lbcb-sci/racon#73]
Hi!

I have long cDNA reads which I want to use as input for genome polishing, expecting that they will only fix exons across the draft reference. Have you tested racon with cDNA alignments before?

I was wondering about how racon deals with "N" operators (CIGAR) of SAM/PAF alignments, as these are suggested to represent introns on mRNA-to-genome alignment. With this in knowledge, do you think it is capable to differentiate between real introns (which are not intended to be removed) and insertions on draft's exons (which should be polished)?

**Comment by rvaser** [comment]
Hello Salvador,
unfortunately, I do not recall evaluating with RNA sequences during development. The N operations will be treated as deletions.

Best regards,
Robert

**Comment by SalvadorGJ** [comment]
Hi Robert,

Thanks for your answer. Indeed I was expecting that. Meanwhile I found a solution using [SplitNCigarReads](https://gatk.broadinstitute.org/hc/en-us/articles/13832774383643-SplitNCigarReads) from GATK suite. This tool divides the splicing cDNA BAM alignments into "only exonic part" alignment, by getting rid of introns. Then I used Racon using the parameters -u (Output unpolished target sequences) and --no-trimming (Disables consensus trimming at window ends). I tried once and seemed to work fine, but I didn't have the time to evaluate it exhaustively.

I leave my comment hoping that it can be of help to someone in the future, as well as to receive your feedback if you think that the use of the parameters is appropriate.

Best,
Salvador
